.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "earthoceanfarms"]
 	path = earthoceanfarms
 	url = git@git.vauxoo.com:vauxoo/earthoceanfarms.git
-	branch = 16.0
+	branch = 16.0-add_l10n_mx_edi_discount-module-CLaurelB
```

Comment:
Remember return this to the stable branch before the merge.

Please set this PR to draft